### PR TITLE
chore: setup typescript dev-env for svelte-flicking

### PR DESCRIPTION
**Pull request by WoodNeck**
## Details
This adds some build scripts for svelte-flicking to use typescript-based codes in svelte-flicking.
By calling `npm run build`, it will produce a folder named `lib` with transpiled codes of ts/svelte sources.

### CODE REVIEW

**Review by mixed — APPROVED** [review]
LGTM

File: packages/svelte-flicking/src/FlickingPanel.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/*
```
Comment (daybrush) [inline packages/svelte-flicking/src/FlickingPanel.js:1]: d.ts files are available in `flicking-panel.svelte.d.ts` format.

Comment (WoodNeck) [reply packages/svelte-flicking/src/FlickingPanel.js:1]: I'll try that 👍

**Review by daybrush — APPROVED** [review]
LGTM

**Review by ahn3407 — APPROVED** [review]
LGTM

**Pull request merged**